extensions/ql-vscode/src/remote-queries/run-remote-query.ts

```diff
@@ -39,7 +39,7 @@ interface QueriesResponse {
     invalid_repositories?: string[],
     repositories_without_database?: string[],
   },
-  repositories_queried?: string[],
+  repositories_queried: string[],
```

Comment:
Note: We originally marked this property as optional, for backwards compatibility with a previous version of the API. The API has been in the current state for long enough that we can mark it as required! 

(If no repos were able to be queried, then the API call returns an error, and we get no `QueriesResponse` data.)